posts/2023/10/hpy_meetup.md

```diff
@@ -0,0 +1,136 @@
+<!--
+.. title: HPy meetup and CPython core dev sprint in Brno (Oct 7-10, 2023)
+.. slug: hpy-meetup-2023
+.. date: 2023-10-10 10:00:00 UTC
+.. author: fangerer
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+We just ended our rather informal HPy meetup in Brno (Czech Republic) which
+started on Saturday (October 7th) and lasted until Tuesday (October 10th).
+
+The main topics of the HPy meetup were:
+
+- The CPython Core Dev Sprint
+- Cython/HPy
+
+<!--TEASER_END-->
+
+# The CPython Core Dev Sprint
+
+Petr Viktorin, one of the most active CPython core developers, invited us to
+partially attend the CPython core developer sprint. As a part of the sprint, he
+organized a C API Summit where we were invited and happily attended this event.
+
+Besides of lots of discussions, there were also very interesting presentations
+about the C API:
+
+## 1. C API Work Group Summary
+
+Irit Katriel presented the current status of the [C API Work Group
+GitHub](https://github.com/capi-workgroup) organisation. The idea of this GitHub
+org was to gather curent [problems](https://github.com/capi-workgroup/problems)
+of the C API and also to collect ideas for the (re-)volution of the API. Irit
+thankfully went over all issues and discussions to structure and summarize them.
+She identified nine categories for problems:
+
+1. API Evolution and Maintenance
+2. API Specification and Abstraction
+3. Object Reference Management
+4. Type Definition and Object Creation
+5. Error Handling
+6. API Tiers and Stability Guarantees
+7. The C Language
+8. Implementation Flaws
+9. Missing Functionality
+
+At this point, I also want to mentioned the strengths of the C API which we also can
+agree on:
+
+- Heap types
+- APIs taking a C string to look up based on Python strings
+- Limited API and Stable ABI
+
+Now, the idea is to write an _Informational PEP_ and then to create:
+
+1. A process for discussions about replacement APIs
+2. Guidelines for [evolution of the C API](https://github.com/capi-workgroup/api-evolution).
+
+## 2. Petr's Vision for the C API
+
+Petr presented his understanding of how the C API should look in
+future. Many of the concepts he presented are very similar to core concept of
+HPy. Maybe the HPy project could positively influence him (since Petr also
```

Comment:
```
Maybe the HPy project could positively influence him (since Petr also
regularly participates in the HPy dev call meetings) but who knows?
```

I would remove the entire sentence, it doesn't add anything.